crates/sui-core/src/authority.rs

```diff
@@ -2975,6 +2976,10 @@ impl AuthorityState {
             cur_epoch_store.epoch()
         );
 
+        if cfg!(debug_assertions) {
+            cur_epoch_store.check_all_executed_transactions_in_checkpoint();
```

Comment:
I forgot but could you double check if we remove tx from the epoch executed table when we revert tx?

Reply:
ah, good point, i don't think we do.

Reply:
fixed - also had to switch to using `executed_transactions_to_checkpoint`, since that is populated by executer and is thus guaranteed to be complete.